sense/finetuning.py

```diff
@@ -92,8 +92,8 @@ def __getitem__(self, idx):
                 position = np.random.randint(minimum_position, num_preds - self.num_timesteps)
                 features = features[position: position + self.num_timesteps]
             # will assume that we need only one output
-        if temporal_annotation is None:
-            temporal_annotation = [-100]
+        if temporal_annotation is None or len(temporal_annotation) == 0:
+            temporal_annotation = np.array([-100])
```

Comment:
Why was adding `len(temporal_annotation) == 0` needed here? When does that happen?
Will this lead to ignore videos that were annotated but didn't contain non-background tags?

Reply:
> Will this lead to ignore videos that were annotated but didn't contain non-background tags?

I guess not... Had a quick look and I think `temporal_annotation` is full of 0s in that case. If that's correct, then you can ignore my comment above ;)

Reply:
When committing this recently I also wasn't sure, which case this was added for, but at the time of the hackathon I'm pretty sure it crashed exactly there. Leaving it in shouldn't hurt, right? :shrug: